charts/fybrik/files/taxonomy/taxonomy.json

```diff
@@ -15,9 +15,33 @@
     "AppInfo": {
       "type": "object"
     },
+    "Authentication": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        }
```

Comment:
what about namespace? We need both to access to a secret

Reply:
don't think authentication have to be a secret. That is one option.

Reply:
@shlomitk1 In taxonomy `name` is the common name for a discriminator field (like in a Connection that name would be `s3`, `db2`, etc.). 

@rohithdv I don't get this one. The example taxonomy you added does not match this definition at all.

Reply:
@roee88 Updated the taxonomy.json to have the output of example taxonomy. Please review. Thanks.

Reply:
sorry I didnt understand..are you referring to the `Authentication` field ? If yes, then in the layers `authentication.yaml` file, do you mean we should have `oneOf` construct like we have for the `Connection` field ?

Reply:
sorry I didnt understand. Are you referring to `Authentication` field? If yes, then do you mean that we have to use the `oneOf` construct like we do for the `Connection` field in the layers file ? @roee88

Reply:
Updated assetCredentials to have vaultPluginPath as is currently used in the protos as discussed with Revital. Please review. Thanks. @roee88 @revit13